### ISSUE TITLE
Arc distances as an float ??

### ISSUE THREAD

**Issue by erikpa1** [RyanCarrier/dijkstra#23]
Hi, it could be nice to have distance of edges as float type. The best could be to be able to choose my own data type, which is probably hard in golang, but maybe, dispatching from OOP concept and go back fully functional to make it easier ?

**Comment by RyanCarrier** [comment]
Hey Mate sorry for not getting to this

Just want to keep this project super simple.
Depending on the scale of your floats, x1000 and putting them as int is probably easiest, otherwise claude could change the distances to floats in about 10s!

Going to leave this as if but obviously feel free to fork :)

**Comment by erikpa1** [comment]
@RyanCarrier  yes I created wrapper  with x100  :) anyway thank for this library, have a nice day :)
